cglib/src/main/java/net/sf/cglib/core/AbstractClassGenerator.java

```diff
@@ -75,6 +75,9 @@ public Object apply(AbstractClassGenerator gen) {
         };
 
         public ClassLoaderData(ClassLoader classLoader) {
+            if (classLoader == null) {
```

Comment:
Current cglib assumes class loader is not null. Otherwise it will fail with NPE here: https://github.com/cglib/cglib/blob/ef8fdd49bc71932486d4bd2d489414f8452f579a/cglib/src/main/java/net/sf/cglib/core/ReflectUtils.java#L409